bin/render-activity-stream-html.js

```diff
@@ -97,7 +97,7 @@ for (const src of ${JSON.stringify(scripts, null, 2)}) {
 <html lang="${options.locale}" dir="${options.direction}">
   <head>
     <meta charset="utf-8">
-    <meta http-equiv="Content-Security-Policy" content="default-src 'none'; script-src 'unsafe-inline' resource: chrome:; connect-src https:; img-src https: data: blob:; style-src 'unsafe-inline';">
+    <meta http-equiv="Content-Security-Policy" content="default-src 'none'; script-src resource: chrome:; connect-src https:; img-src https: data: blob:; style-src 'unsafe-inline';">
```

Comment:
Looks good!  :+1: 

The adjacent `connect-src` surprises me a bit. Do you actually use fetch, XHR, EventSource, sendBeacon,   `<a ping>` or Websockets?
To clarify, I found it just surprising. Not concerning :)

Reply:
Ah good catch – we used to use `fetch` from old snippets, but it's no longer needed. I supposed we can remove that

Reply:
Nevermind, still needed for one of the form submit actions in a template.